rasa/core/training/dsl.py

```diff
@@ -189,10 +187,11 @@ def __init__(self, domain, interpreter, template_vars=None, use_e2e=False):
 
         # if exclusion percentage is not 100
         if exclusion_percentage and exclusion_percentage != 100:
-            import random
+            import numpy as np
 
+            np.random.seed(42)
```

Comment:
this is different usage, it is meant to be comparison script that is run several times, so we specifically didn't fix random seed here

Reply:
Okay, correct me if i am wrong but this is being used in rasa/core/train.py at line 123. If exclusion_percentage is provided it will start generating random story_steps and will result in random training_data

Reply:
yes, if exclusion_percentage is provided, randomness is intended

Reply:
okay, got it. reverting